src/sql/finalize.sql

```diff
@@ -346,6 +346,20 @@ STRICT LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_pgvectors_upgrade_wrapper';
 CREATE FUNCTION to_svector(dims INT, indices INT[], vals real[]) RETURNS svector
 IMMUTABLE STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_to_svector_wrapper';
 
+CREATE FUNCTION text2vec_openai(input TEXT, model TEXT) RETURNS vector
+STRICT LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_text2vec_openai_wrapper';
+
+CREATE FUNCTION text2vec_openai_v3(input TEXT) RETURNS vector
```

Comment:
I still thinks `text2vec_openai_3small` is better. God knows if OpenAI will release `text-embedding-3-turbo-small` or `text-embedding-3-very-small`.

Reply:
We could remind and track it when further update.